### DISCUSSION TITLE
Cannot set env variable TRUSTED_PROXIES for docker container.

### DISCUSSION THREAD

**Discussion by mdotkurmi** [firefly-iii/firefly-iii#5935] [Firefly III] [answered]
I am trying install firefly on docker. As I will run this behind NGNIX, I am trying to set the variable TRUSTED_PROXIES=** as suggested in other fixes.
I get the following error as shown in the image below.
No matches found for TRUSTED_PROXIES=**
Please help to fix this.

**Comment by JC5** [comment, marked as answer]
It's expanding the variable. Try `-e 'TRUSTED_PROXIES=**'`

**Reply by mdotkurmi** [reply]
Thanks alot.
It worked!
